zip-proportional-output-fee-mechanism-pofm.rst

```diff
@@ -0,0 +1,195 @@
+::
+
+  ZIP: XXX
+  Title: Proportional Output Fee Mechanism (POFM)
+  Owners: Aditya Bharadwaj <nighthawk24@gmail.com>
+  Credits: Madars Virza, Kris Nuttycombe
+  Status: Draft
+  Community Status: Request for comments : https://forum.zcashcommunity.com/t/zip-proportional-output-fee-mechanism-pofm/42808
+  Category: Wallet
+  Created: 2022-08-15
+  License: MIT
+
+
+Terminology
+===========
+
+The key words "MUST", "SHOULD", "SHOULD NOT", "MAY", "RECOMMENDED",
+"OPTIONAL", and "REQUIRED" in this document are to be interpreted as
+described in RFC 2119. [#RFC2119]_
+
+"Z" refers to shielded address.
+"T" refers to transparent address.
+
+"POFM" refers to the proposed change in conventional fee as described in 
+this ZIP.
+
+The term "default transaction fee" in this document is in reference
+to Z to Z, T to Z & Z to T transaction fee paid to miners on the Zcash network
+for the work on including the shielded transaction in a block.
+
+
+Abstract
+========
+
+The goal of this ZIP is to change the conventional transaction fees for
+Shielded transactions and get buy-in from wallet developers, miners & Zcash users.
+
+With an updated transaction fee formula, miners will be compensated fairly for
+including transactions with a high number of outputs, while still allowing low fees 
+for regular shielded transaction use cases.
+
+
+Out of Scope for this Proposal
+============
+
+Discussion around transaction fees for T to T.
+
+
+Motivation
+============
+
+In light of recent network activity, it is time to review and update the 
+standard 1,000 zats transaction fees set in ZIP-313.
+
+
+The conventional transaction fee presently is 0.00001 ZEC or 1,000 zats per
+ZIP-313, that allowed exploration of novel use cases of the Zcash blockchain.
+The Zcash network has operated for almost 2 years of 1,000 zats fee per shielded 
+transaction without consideration for the total number of inputs and outputs in the transaction.
+This has resulted in high output transactions with 1,100 outputs costing the same as 
+transactions with 2 outputs.
+
+
+Requirements for consensus
+-------------------------
```

Comment:
One note: this is not actually a consensus change; it impacts transaction relay (which is part of the network protocol) and the default selection mechanism used by miners (and, as it is not a change to consensus, does not require a hardforking upgrade.)

Reply:
Correct, the wording is targeted towards gathering consensus with software developers for adopting the change.